reana_db/utils.py

```diff
@@ -13,16 +13,25 @@
 from sqlalchemy import inspect
 
 
-def build_workspace_path(user_id, workflow_id=None):
+def build_workspace_path(user_id, workflow_id=None, workspace_root_path=None):
     """Build user's workspace relative path.
 
     :param user_id: Owner of the workspace.
     :param workflow_id: Optional parameter, if provided gives the path to the
         workflow workspace instead of just the path to the user workspace.
-    :return: String that represents the workspace relative path.
-        i.e. users/0000/workflows/0034
+    :param workspace_root_path: Optional parameter, if provided changes the
+        default directory where the workflow is stored.
```

Comment:
Please amend doctring based on changes in pytest-reana.

Please add yourself to the AUTHORS list.

Please add a release commit on top.